WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java

```diff
@@ -107,4 +109,28 @@ private void clickItem(final Matcher<View> itemViewMatcher) {
                     .perform(actionOnItem(hasDescendant(itemViewMatcher), click()));
         }
     }
+
+    public static void goToPosts() {
+        goToMenuTab();
```

Comment:
@ovitrif - love this improvement, as it makes it very clear that we have to navigate to the menu tab first. ❤️

Reply:
Thank you 🎉 💯 🚀